modules/oc-mirror-enclave-support-about.adoc

```diff
@@ -19,4 +19,35 @@ Enclave support has the following benefits:
 
 * You can mirror content directly from the centralized internal registry to enclaves without restarting the mirroring process from the internet for each enclave.
 
-* You can minimize data transfer between network stages, so to ensure that a blob or image is transferred only once from one stage to another.
\ No newline at end of file
+* You can minimize data transfer between network stages, so to ensure that a blob or image is transferred only once from one stage to another.
+
+== How to mirror to an enclave
+
+image::445_OpenShift_Enclave_support_0724.png[Enclave Support]
+
+The above illustration outlines the flow for using the oc-mirror plugin in different environments, including environments with and without an internet connection.
+
+*Environment with Internet Connection*:
+
+- The user executes oc-mirror plugin V2 to mirror content from an online registry to a local disk directory.
+- The mirrored content is saved to the disk for transfer to offline environments.
+
+*Airgapped Enterprise Environment (No Internet)*:
+
+-  Flow 1:
```

Comment:
So I am not sure whether Flow 1 and Flow 2 are two different options that users can choose to use, or something that users have to do in order. If they both need to be done, and in order, I would make this line and [line 41](https://github.com/openshift/openshift-docs/pull/81930/files#diff-50b8a98363a7e5aa947d524510775ee4db48b65b89c6206ffa7bee64678c609aR41) a numbered list item.

If they are not flows that need to be done in order, then they can stay as unordered list items, but I would suggest replacing the hyphen with a * instead. This would just follow the way our docs typically do things a bit more closely:
```suggestion
*  Flow 1:
```